### ISSUE TITLE
Comparisons to TensorFlow and pytorch impls make less sense than comparisons to onnxruntime imp,s

### ISSUE THREAD

**Issue by KOLANICH** [OpenNMT/CTranslate2#1023]
In my experience (though, for other models, CNN and visual transformer ones) onnxruntime provides better performance for inference on CPU than pytorch, tensor and (surprisingly) Apache TVM (CPU and Vulkan GPU backends, the models were "optimized"). So it may make little sense to compare CTranslate2 (which is targeted for fast inference) to framework targeted for convenient model design and training.

**Comment by guillaumekln** [comment]
Yes, it would be great to compare against onnxruntime, but I haven't explore it at the moment. We should find a model that can be run with both CTranslate2 and onnxruntime because we should compare performance for a fixed quality.

Note that the benchmark does not only include TensorFlow and PyTorch. There are numbers from [FasterTransformer](https://github.com/NVIDIA/FasterTransformer) and [Marian](https://github.com/marian-nmt/marian) which also focus on performance.

**Comment by KOLANICH** [comment]
TF and especially pytorch models can be converted into ONNX format. So I guess one can take a pytorch model and export it as ONNX. I did so with Apple MobileViT model and it has given identical vectors when inferred via pytorch and onnxruntime.

**Comment by vince62s** [comment]
so you're welcome to PR on OpenNMT-py :)

**Comment by guillaumekln** [comment]
I know that ONNX works great for vision models, but translation models are often a different story. As far as I know it's still quite tricky to convert and execute encoder-decoder models that are using beam search.

The most promising end-to-end integration of onnxruntime is probably https://github.com/huggingface/optimum, but I just tried a translation with the model `Helsinki-NLP/opus-mt-en-de` and it was slower than PyTorch out-of-the-box. I will investigate when I have some time.
